### ISSUE TITLE
"Examples" link in the README is broken when viewed from hub.docker.com

### ISSUE THREAD

**Issue by jcoyne** [lambci/docker-lambda#96]
See here: https://hub.docker.com/r/lambci/lambda/

**Comment by tdmalone** [comment]
Yes it would, but then it would link to the wrong file in forks.

I guess it depends if you want it to work more on GitHub... or external services. I tend to think it should link relatively, so that the repo name isn't hardcoded, and the service displaying the file should work out how to handle that. But that's just my opinion and this isn't my repo :P
